phys2bids/interfaces/mat.py

```diff
@@ -0,0 +1,72 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+
+"""phys2bids interface for MATLAB files."""
+
+import logging
+
+import numpy as np
+from scipy.io import loadmat
+
+from phys2bids.physio_obj import BlueprintInput
+
+LGR = logging.getLogger(__name__)
+
+
+def populate_phys_input(filename, chtrig):
+    """
+    Populate object phys_input from MATLAB files.
+
```

Comment:
```suggestion
    Populate object phys_input from Labchart files in MATLAB format.

```

Reply:
Actually, I think that the next lines deal with AcqKnowledge import as well?

Reply:
Ok, after a private chat with Eneko I got what's happening. Reverting this PR to draft!